ansible/files/creategraph.py

```diff
@@ -46,7 +48,7 @@ def __init__(self, dev_csvfile=None, link_csvfile=None, cons_csvfile=None, pdu_c
         self._cache_port_alias_to_name = {}
         self.pngroot = etree.Element('PhysicalNetworkGraphDeclaration')
         self.dpgroot = etree.Element('DataPlaneGraph')
-        self.csgroot = etree.Element('ConsoleGraphDeclaration')
```

Comment:
I have concern on this naming change, i think it will break some existing things.
Can we create a new one without touching existing one? The connection graph is layered, we can duplicate device into different graph if they have mutiple capability